src/views/Rename.vue

```diff
@@ -174,15 +180,47 @@ export default class Rename extends Vue {
     .wallet-label {
         display: flex;
         flex-direction: row;
-        justify-content: flex-start;
+        justify-content: center;
         align-items: center;
         font-size: 2.25rem;
         line-height: 2.5rem;
         font-weight: 500;
-        margin: 2rem 3rem 0;
+        margin: 0 3rem;
         padding: 2rem 1rem;
     }
 
+    :global(.label-input) {
```

Comment:
What does `:global` do? Is it like using `>>>`?

Reply:
Similar. But I honestly forgot about `>>>`. I have refactored the styles to use `>>>` instead.